### PR TITLE
Updated all sub-tools to riemann-tools 1.1.1 and increment versions

### PR DESCRIPTION
**Pull request by jamtur01**
Signed-off-by: James Turnbull <james@lovedthanlost.net>

### CODE REVIEW

**Comment by smortex** [comment]
Anything blocking I can help with?

**Comment by jamtur01** [comment]
@smortex The big blocker is mostly time. I've tested some of the tools but others are harder without infrastructure in place. Additionally, it is unclear if some of them have worked for a while, for example the `riemann-aws-status` tool has likely been broken a while. You're welcome to dive in and start testing individual tools.

**Comment by jamtur01** [comment]
I'll leave this branch as this - I think fixing anything else should be in other branches before this one becomes unfit for purpose.

**Comment by smortex** [comment]
Hehe, I don't use any of these tools except form ElasticSearch.  Time to give it a try :wink:

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,146 @@
+PATH
```
Comment (smortex) [inline Gemfile.lock:1]: Do we really want to include the `Gemfile.lock` in the repo?  For a gem like this one that looks odd to me…

Comment (jamtur01) [reply Gemfile.lock]: Odd. I thought I had excluded it. Will remove.

**Comment by jamtur01** [comment]
I am going to merge this - I am sure other things are broken but let's burn those bridges when we get to them,

**Pull request merged**